src/App.js

```diff
@@ -11,7 +11,7 @@
 
 import React from 'react'
 
-import GameBoard from './GameBoard'
+import GameBoard from './components/GameBoard'
```

Comment:
Off topic trick. You can achieve absolute path import by using jsconfig.json
https://saurabhshah23.medium.com/react-app-with-absolute-paths-using-jsconfig-json-2b07b1cb24d4